private/registry-names-update.sps

```diff
@@ -0,0 +1,49 @@
+;; Copyright 2019 Lassi Kortela
+;; SPDX-License-Identifier: MIT
+
+(import (chezscheme))
+
+(define output-file "registry-names.sls")
+
+(define download-command
+  (string-append
+   "curl --fail --silent --show-error "
+   "https://raw.githubusercontent.com/"
+   "scheme-requests-for-implementation/srfi-common"
+   "/master/"
+   "admin/srfi-data.scm"))
+
+(define transcoder (make-transcoder (utf-8-codec) (eol-style lf)))
+
+(define (read-all port)
+  (let loop ((xs '()))
+    (let ((x (read port)))
+      (if (eof-object? x) (reverse xs) (loop (cons x xs))))))
+
+(define provided
+  (call-with-port (open-input-file "registry-provided.scm") read-all))
+
+(define srfi-data
+  (let-values (((to-process from-process process-stderr process-id)
+                (open-process-ports download-command 'block transcoder)))
+    (read-all from-process)))
+
+(define mapping
+  (map (lambda (srfi)
+         (list (cadr (assoc 'number srfi))
+               (cadr (assoc 'library-name srfi))))
+       (filter (lambda (srfi)
+                 (and (assoc 'library-name srfi)
+                      (member (cadr (assoc 'number srfi)) provided)))
+               srfi-data)))
```

Comment:
This is the part where the filtering is done. If we remove the `(member ...)` expression then the list will include all SRFIs, even those that are not in chez-srfi.